tests/lang_callable/template/template_issues.nim

```diff
@@ -296,3 +296,11 @@ block: # bug #12595
     discard {i: ""}
 
   test()
+
+block: # https://github.com/nim-works/nimskull/issues/1015
+  proc takeInt(i: int) = discard
+
+  template x(b: static[bool] = false): untyped =
+    takeInt b.int
+
+  x()
```

Comment:
```suggestion
block static_parameter_with_default:
  # using a `static` template parameter as the argument to a conversion
  # crashed the compiler
  template x(b: static[bool] = false) =
    discard int(b)

  x() # the default value for the parameter must be used
```
The intention is to only use the language features that are necessary and to document highlight the important parts, making navigating the test easier.